ihatemoney/templates/layout.html

```diff
@@ -65,9 +65,7 @@ <h1><a class="navbar-brand" href="{{ url_for("main.home") }}"><span>#!</span> mo
           <div class="dropdown-menu" aria-labelledby="langMenuButton">
               <h6 class="dropdown-header">{{ _('Languages') }}</h6>
               {% for lang in config['SUPPORTED_LANGUAGES'] %}
-                {% if g.lang != lang %}
                   <a class="dropdown-item" href="{{ url_for("main.change_lang", lang=lang)}}">{{ locale_from_iso(lang).display_name | capitalize }}</a>
-                {% endif %}
```

Comment:
What about writing a mark in front of the current language?

```html
<a class="dropdown-item" href="{{ url_for("main.change_lang", lang=lang)}}">
  {% if g.lang == lang %}
  <span class="select-mark">•</span>
  {% endif %}
  {{ locale_from_iso(lang).display_name | capitalize }}
</a>
```

And in CSS we need:
```css
.dropdown-item > .select-mark {
  margin-left: -14px;
  margin-right: 4px;
}
```

Rendered:
![Screenshot_2021-07-18 Gestion de comptes - demonstration](https://user-images.githubusercontent.com/297578/126050743-dc8c444d-7222-4a3e-bf11-2aaa9d414762.png)

I chose an Unicode character, but feel free to suggest anything else, it's just the idea.

Reply:
Right, good idea! I've simply added the "active" bootstrap class:

![2021-07-18-143017_274x784_scrot](https://user-images.githubusercontent.com/731144/126067126-7d25e95d-b069-4efb-a9ac-1ca5edc5cf23.png)